.github/workflows-src/jobs.lib.yml

```diff
@@ -73,11 +73,13 @@ steps:
 name: "Integration Test"
 strategy: *strategy
 runs-on: *runs-on
+needs: [ prepare ]
```

Comment:
Do we still need the other changes, when integration test jobs are dependent on prepare?